### DISCUSSION TITLE
qtile with ultrawide monitors

### DISCUSSION THREAD

**Discussion by jobidon** [qtile/qtile#4030] [General]
I'm looking for a tiling window manager that can help me with my setup. Let let me know if/how you think qtile could be applied to the following scenario.

I have a single huge flat (not curved) 4k monitor. The monitor is so big that I barely look at the top left/right corners and in fact spend most of my time looking at the bottom center of the screen where I swap windows in/out of that area as I work on them. I'd like to separate the display in 6 different zones (1..6) where the master zone where I focus my work is zone 5.

``` 
    25%             50%            25%
 _______________________________________
|          |                   |         |
|     1    |         2         |    3    |
|   Dash   |    Live Charts    |   IM    |     50%
|          |                   |         |
|__________|___________________|_________|
|          |                   |         |
|    4     |         5         |    6    |
|   Spawn  |      Master       |  Tasks  |     50%
|          |                   |         |
 __________|___________________|_________|
```
Column 1 (zone 1 and 4) takes up 25% of the width of the screen. Same thing for column 3 (zone 3 and 6), leaving the middle section with 50% of the screen. The status/menu bar is at the bottom of the screen, below all zones.

Most tiling manager have a way to make that grid layout work fine, but the next part is where things get tricky.

I currently have to manually move a window to zone 5 when I want to focus on it for more than a few seconds. I currently do that with the mouse and have to resize as best I can. And when I'm done, I have to move it back to where it was before the focus occured. I'd like to create a shortcut to move/resize a window to zone 5 and at the same time move the window currently in zone 5 to another zone (I could use zone 4 as a place to spawn new window and temporarily place unfocused windows). 

So is this something qtile can be customized to do? What are your suggestions regarding how to do this?

thanks for your input!

**Comment by elParaguayo** [comment]
You could use `fake_screens` to split your screen into multiple areas with each area being able to display a different group.
http://docs.qtile.org/en/stable/manual/config/screens.html?highlight=fake_screens#fake-screens

**Comment by jobidon** [comment]
Thanks for your insight. 
If I'm understanding correctly, I would need to define 6 virtual screens using `fake_screens`, create 6 `groups`, and assign a group to each screen. And create a `layout` for each group to define how windows would be displayed in each screen.
Then I need to create a keybinding to `cmd_toscreen()` each window I want to focus on, and, if needed, to it for any window already in the master screen.
Does that sum it up accurately?

**Reply by elParaguayo** [reply]
Yes, that's pretty much it.

One issue that you may run into is that, if you try to focus a group on a different screen, that group will get pulled to the current screen so you need to change the screen focus first.

That issue is discussed here: http://docs.qtile.org/en/stable/manual/faq.html#how-can-i-get-my-groups-to-stick-to-screens
